.envrc

```diff
@@ -25,11 +25,12 @@ export FISSILE_RELEASE="${FISSILE_RELEASE#,}"
 export FISSILE_ROLE_MANIFEST="${PWD}/role-manifest.yml"
 export FISSILE_LIGHT_OPINIONS="${PWD}/opinions.yml"
 export FISSILE_DARK_OPINIONS="${PWD}/dark-opinions.yml"
-export FISSILE_WORK_DIR="${PWD}/work-dir"
+export FISSILE_WORK_DIR="$HOME/.fissile/uaa-work-dir"
```

Comment:
That wouldn't exist if you're developing this locally (outside of any vagrant box type things).
Can we make this fallback instead?
```bash
export FISSILE_WORK_DIR="${FISSILE_WORK_DIR:-$HOME/.fissile/uaa-work-dir}"
```
Then we can set a work dir outside, or something.
Otherwise we can make the `.envrc` smarter and check if this looks like a vagrant box and whatnot, but that might be too much.

Actually, why was this changed in the first place?